peachjam/models/generic_document.py

```diff
@@ -85,6 +86,13 @@ class Meta(CoreDocument.Meta):
     def __str__(self):
         return self.title
 
+    def apply_labels(self):
+        if self.repealed:
```

Comment:
don't forget to call `super().apply_labels()`

Reply:
you also need to handle the case where the label is there incorrectly, and should be removed.